tests/examples/ordering_pagination.feature.example

```diff
@@ -0,0 +1,11 @@
+Feature: Ordering and Pagination
+
+  Users should be able to change the number of entries per page for results.
+  Users should also be able to reorder results by relevance, ascending date, descending date.
+
+  Scenario: reordering from author search
+    Given surname is "cat*"
```

Comment:
Surname *field* I assume?

Reply:
Oh yeah. That too. I'll check throughout, was probably inconsistent elsewhere in the files. :)